docs/advanced/authentication.md

```diff
@@ -309,8 +309,6 @@ const willAuthError = ({ operation, authState }) => {
         );
       })
     );
-  } else if (false /* JWT is expired */) {
```

Comment:
judging from the comment it would be better to do something like `else if (isJwtExpired(authState))`

Reply:
ohh ya, my mistake. I think both the original snippet and your change makes sense.